lib/Repeat_Ride.dart

```diff
@@ -90,32 +188,46 @@ class _RepeatRideState extends State<RepeatRide> {
                 Text('This ride repeats every:', style: descriptionStyle),
               ],
             ),
-            SizedBox(height: 20.0),
-            Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: <Widget>[
-                ToggleButtons(
-                  constraints: BoxConstraints.expand(width: 70, height: 60.0),
-                  color: Colors.grey,
-                  selectedColor: Colors.black,
-                  renderBorder: false,
-                  fillColor: Colors.white,
-                  splashColor: Colors.white,
-                  children: <Widget>[
-                    Text('M', style: toggleStyle),
-                    Text('T', style: toggleStyle),
-                    Text('W', style: toggleStyle),
-                    Text('Th', style: toggleStyle),
-                    Text('F', style: toggleStyle)
-                  ],
-                  onPressed: (int index){
-                    setState(() {
-                      isSelected[index] = !isSelected[index];
-                    });
-                  },
-                  isSelected: isSelected,
-                ),
-              ],
+            SizedBox(height: 10.0),
+            SafeArea(
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  ToggleButtons(
+                    constraints: BoxConstraints.expand(width: 60, height: 60.0),
+                    color: Colors.grey,
+                    selectedColor: Colors.black,
+                    renderBorder: false,
+                    fillColor: Colors.white,
+                    splashColor: Colors.white,
+                    children: <Widget>[
+                      Text('M', style: toggleStyle),
+                      Text('T', style: toggleStyle),
+                      Text('W', style: toggleStyle),
+                      Text('Th', style: toggleStyle),
+                      Text('F', style: toggleStyle)
+                    ],
+                    onPressed: (int index) {
+                      setState(() {
+                        isSelected[index] = !isSelected[index];
+                        selectedDays = setSelectedDays(selectedDays);
+                      });
+                    },
+                    isSelected: isSelected,
+                  ),
+                ],
+              ),
+            ),
+            Form(
+              key: _formKey,
+              child: Column(
+                children: <Widget>[
+                  SizedBox(height: 10.0),
+                  _buildPickupTimeField(),
+                  SizedBox(height: 20.0),
+                  _buildDropOffTimeField(),
```

Comment:
I don't think the pickup and dropoff time fields are shown again on the Repeating Rides page in the design?

Reply:
Indeed, I believe the designs have changed. Now it is dates instead of time.